terasoluna-gfw-functionaltest-parent/pom.xml

```diff
@@ -28,6 +28,27 @@
         <name>TERASOLUNA Framework Team</name>
         <url>http://terasoluna.org</url>
     </organization>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <artifactId>maven-war-plugin</artifactId>
+                    <configuration>

```

Comment:
- [ ] Why this config must be in ``parent`` as ``<pluginManagenemt>`` ? It can be put in ``web`` as ``<dependency>`` and it is more natural because this config is for ``web`` module rather than general one.